portal-web/test/functional/com/liferay/portalweb/macros/OneDrive.macro

```diff
@@ -214,6 +214,17 @@ definition {
 		MenuItem.clickNoError(menuItem = "Edit in Office 365");
 	}
 
+	macro gotoEditInPreviewPage {
```

Comment:
Is possible to combine this macro with `OneDrive.gotoEdit`?Just add a condition to the macro.